README.md

```diff
@@ -138,7 +143,7 @@ When using Odin's [`Injectable` decorator](#injectable-decorator), the injectabl
 
 This snippet shows how to register an injectable manually:
 ```typescript
-import { odin } from '@odinjs/odin';
+import { odin } from '@philips-software/odin';
 
 class Injectable { }
 
```

Comment:
I think `bundle.register(Injectable)` can also be deprecated. Therefore, we have two option for the snippet below.

1. Keep it as it is, it just points out the deprecation through a note.
2. Change it to already register using the `options` param along with the `class` name.